dronegen/windows.go

```diff
@@ -23,6 +23,9 @@ const (
 	toolchainDir      = `/toolchains`
 	teleportSrc       = `/go/src/github.com/gravitational/teleport`
 	webappsSrc        = `/go/src/github.com/gravitational/webapps`
+
+	relcliUrl    = `https://cdn.cloud.gravitational.io/relcli-windows-v1.1.70-justinas.1.exe`
```

Comment:
These are placeholder values (a release created manually on staging) while we work out the relcli windows publishing pipeline on the Cloud side.